modules/nf-core/endorspy/meta.yml

```diff
@@ -20,14 +20,15 @@ input:
       description: |
         Groovy Map containing sample information
         e.g. [ id:'test', single_end:false ]
-  - stats:
+  - stats_raw:
       type: file
-      description: output of samtools flagstat in a text file
-  - stats_optional:
+      description: output of samtools flagstat in a txt file, assumes no quality filtering nor duplicate removal performed
+  - stats_qualityfiltered:
       type: file
-      description: output of a second samtools flagstat run in a text file. If two files are supplied,
-        the mapped reads of the second file is divided by the total reads in the first, since it
-        assumes that the <samplefile.stats> are related to the same sample. Useful after BAM filtering.
+      description: output of samtools flagstat in a txt file, assumes some form of quality or length filtering has been performed, must be provided with at least one of the options -r or -d
```

Comment:
Should we add a check in `script` that if `stats_qualityfiltered && !optionalraw && !deduplicated` it fails?

Reply:
it does fail

Reply:
Sorry I mean we should add a warning if that conditio nis detected

Reply:
You mean in the description?